src/run.js

```diff
@@ -36,13 +36,14 @@ const minimalcss = async options => {
       if (/data:image\//.test(request.url)) {
         // don't need to download those
         request.abort()
-      } else if (/\.(png|jpg|jpeg$)/.test(request.url)) {
+      } else if (/\.(png|jpg|jpeg|webp|gif$)/.test(request.url)) {
```

Comment:
There's a bug in this line. It was there before you made it better :)

https://github.com/peterbe/minimalcss/issues/24

I'm going to try to fix that now. Afterwards, can you rebase up master?

Reply:
If you remove your change and the rebase against master the conflict will go away.